### ISSUE TITLE
[HL2 EP1][Linux] Alyx dissapers between Intervention and lowlife.

### ISSUE THREAD

**Issue by ericwomer** [ValveSoftware/Source-1-Games#693]
After the train crashed in ep1 and alyx is trapped under a stalker, I pulled the stalker off of her then jumped out of the train and ran to the next section when lowlife loaded she was not there. here is my save in lowlife, don't know if its reproducible.

https://dl.dropboxusercontent.com/u/71062164/half-life-2/ep1/half-life-078.sav

**Comment by werybigmonk** [comment]
Problem still exists. At least in one situation I encountered, If a player climbs through the hole in the wall before Alyx jumps out of the train, she won't be in the next map.

**Comment by Kwastie** [comment]
It's still broken :(   

Work around: Esc > New Game > Chapter 3: Lowlife

**Comment by merschformann** [comment]
Just experienced the bug with the windows version.

Christian's workaround still 'solves' the problem. :)

**Comment by jhford** [comment]
This just happened to me on Windows...

**Comment by b789** [comment]
happened just now on the linux version

**Comment by kmishukov** [comment]
Happened to me right now, Windows 10.

**Comment by macmeister1967** [comment]
Can confirm this is still happening  FIVE YEARS LATER... Way to DROP THE BALL Valve !!!

(Ubuntu 16.04 LTS btw).

**Comment by Bangkokian** [comment]
Yep. Happened to me on Nvidia Shield TV. Play-stopping bug. I had no idea Alyx was supposed to be with me all this time. All saves are filled with states where she's already gone. No I don't feel like starting at the beginning and hoping it works out this time.

Nice job Valve. You had how many years to fix this? Not enough cash rolling in?

Game deleted.

**Comment by MysteryFluid** [comment]
I just recreated it after revisiting the game for the first time in 7 years. I pulled the stalker off Alyx and immediately ran outside past the transition point into the next chapter and she disappeared.

**Comment by Hyperbuss** [comment]
just happened to me and I found this thread XD
replaying all of them for the 20th anniversary

**Comment by NorthernChicken** [comment]
Happened to me as well (Linux)

**Comment by knutz0r** [comment]
this bug still exists in 2025, lol. new game c3 low life fixed it

**Comment by lranixonl** [comment]
Yes, still happening

**Comment by danieldormann** [comment]
12 years later, but still a thing (:

**Comment by Grungus12** [comment]
I recently had the same problem on windows 10. I believe what happens is if you jump off the train and go through the hole in the wall too quickly and trigger the "loading" moment Alyx will not spawn in the area with you. you end up soft-locked and have to reload the save and just wait for Alyx to exit the train before triggering the load. I took a recording when I noticed I was soft-locked but do not know the best way to share that here.
